test/integration/build/aws.tf

```diff
@@ -1619,31 +1619,24 @@ resource "aws_lb" "aws-alb" {
   }
 }
 
-resource "aws_cloudformation_stack" "network" {
+resource "aws_cloudformation_stack" "ecr" {
   count = var.aws_enable_creation
   name  = var.aws_cloudformation_stack_name
 
-  parameters = {
-    VPCCidr = "10.0.0.0/16"
-  }
-
   template_body = <<STACK
 {
   "Parameters" : {
-    "VPCCidr" : {
+    "ECRrepo" : {
       "Type" : "String",
-      "Default" : "10.0.0.0/16",
-      "Description" : "CIDR block for the VPC"
+      "Default" : "aws-inspec-cloudformation_stack_ecr_name_fuevqbvfns",
```

Comment:
Can we maybe inject a random string here or if static, make this a parameter we inject instead?